nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
[OPTIONAL] You can improve your error handling by adding a custom message that will specify the class and method from which the error is coming from.

```
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```

Reply:
Hi @KDlamini 👋, 
Thank you for taking the time to review my code and providing helpful feedback. I have made the suggested [OPTIONAL] changes to improve error handling by adding a custom error message that specifies the class and method from which the error is coming from. 🚀 💯